lib/run/src/Obelisk/Run.hs

```diff
@@ -130,11 +131,11 @@ runWidget
   -> IO ()
 runWidget conf configs frontend validFullEncoder = do
   uri <- either (fail . T.unpack) pure $ getConfigRoute configs
-  let port = fromIntegral $ fromMaybe 80 $ uri ^? uriAuthority . _Right . authPort . _Just
+  let port = fromMaybe (fromIntegral $ fromMaybe 80 $ uri ^? uriAuthority . _Right . authPort . _Just) (_runConfig_portOverride conf)
       redirectHost = _runConfig_redirectHost conf
       redirectPort = _runConfig_redirectPort conf
       beforeMainLoop = do
-        putStrLn $ "Frontend running on " <> T.unpack (URI.render uri)
+        putStrLn $ "Frontend running on " ++ T.unpack (URI.render (uri & uriAuthority . _Right . authPort ?~ (fromInteger (fromIntegral port))))
```

Comment:
It doesn't sit right with me that we have to edit both the log message and the actual logic to make them coherent together. I'm not sure what the solution is tho.